Contest_API.md

```diff
@@ -1058,6 +1056,26 @@ clients with `admin` or `analyst` role. The `reaction` attribute
 is available to clients with `public` role only when the contest is
 not frozen.
 
+#### POST submissions
+
+To add submissions one can use the `POST` method on the submissions endpoint.
+The `POST` must include a valid JSON object with the same attributes the submission
+endpoint returns with a `GET` request with the following exceptions:
+
+* Any provided `id`, `reaction` and `contest_time` attributes are ignored.
+* The `time` attribute is optional. If not provided (or `null`) it will default
+  to the current time as determined by the server.
+* Since `files` only supports `application/zip`, providing the `mime` field is
+  optional.
+* If the CCS supports a `team` role, the `team_id` and `time`
+  attributes will be ignored when using a this role. `team_id` will then use
+  the ID of the team associated with the request and `time` will always use
+  the current time as determined by the server.
+
+The response will be the ID of the newly added submission.
+
+The `public` role can never add submissions.
```

Comment:
Maybe better to be more explicit and say that `POST`s by any other roles than `admin` and `team` are not supported.